ouroboros-network/src/Ouroboros/Network/BlockFetch/Client.hs

```diff
@@ -190,6 +190,7 @@ blockFetchClient _version controlMessageSTM
                 Right lower = AF.last fragment
                 Right upper = AF.head fragment
 
+        traceWith tracer (SendFetchRequest (FetchRequest [fragment]))
```

Comment:
Maybe use `AnchoredFragment` as the tracer argument?

Reply:
Great point, by the way!

Reply:
Done!